lib/agenda/every.ts

```diff
@@ -14,7 +15,7 @@ const debug = createDebugger('agenda:every');
  * @param options - options to run job for
  * @returns Job/s created. Resolves when schedule fails or passes
  */
-export const every = async function(this: Agenda, interval: string | number, names: string | string[], data: any, options: any): Promise<any> {
+export const every = async function(this: Agenda, interval: string, names: string | string[], data: any, options: JobOptions): Promise<any> {
```

Comment:
if i remember right, the interval can also be a number

Reply:
I just checked again. "interval" is passed down further down the chain and eventually ends up in JobAttribute.repeatInterval.
This fields needs to be a "human readable" time-interval or cron-expression.